Kickstarter-iOS/Views/Controllers/ProjectNavBarViewController.swift

```diff
@@ -186,10 +186,6 @@ public final class ProjectNavBarViewController: UIViewController {
     self.shareViewModel.outputs.showShareSheet
       .observeForControllerAction()
       .observeValues { [weak self] in self?.showShareSheet($0) }
-
-    self.shareViewModel.outputs.showShareCompose
-      .observeForControllerAction()
-      .observeValues { [weak self] in self?.showShareCompose($0) }
```

Comment:
Can you explain why this was removed? I am forgetting the difference b/w the shareSheet and shareCompose